internal/gen-atomicint/wrapper.tmpl

```diff
@@ -18,6 +18,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:build !go1.19
+// +build !go1.19
```

Comment:
This line is not needed in Go 1.17+

https://pkg.go.dev/cmd/go#hdr-Build_constraints

> Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.